Program.fs

```diff
@@ -130,10 +122,18 @@ let findInstalledPackages (packageJson: string) : ResizeArray<InstalledNpmPackag
             if not (IO.File.Exists pkgJson)
                 then ()
             else
-                let pkgJsonContent = JObject.Parse(File.readAllTextNonBlocking pkgJson)
+                let nameAndVersionDecoder = Decode.object (fun it ->
```

Comment:
Suggestion: use `get` instead of `it` as per @MangelMaxime recommendation because this is a "field getter"